Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs

```diff
@@ -59,19 +60,26 @@ public void OnBeginExecuteCallback(long id, string dataSource, string database,
                 var telemetryTuple = this.TelemetryTable.Get(id);
                 if (telemetryTuple == null)
                 {
-                    bool isCustomCreated = false;
                     var telemetry = ClientServerDependencyTracker.BeginTracking(this.telemetryClient);
                     telemetry.Name = resourceName;
                     telemetry.Target = string.Join(" | ", dataSource, database);
                     telemetry.Type = RemoteDependencyConstants.SQL;
                     telemetry.Data = commandText;
-                    this.TelemetryTable.Store(id, new Tuple<DependencyTelemetry, bool>(telemetry, isCustomCreated));
+                    this.TelemetryTable.Store(id, new Tuple<DependencyTelemetry, bool>(telemetry, false));
```

Comment:
is this boolean value ever NOT false?
I only see one example below of an if statement checking this value.

Reply:
this value was explicitly set to false few lines above, so it has always been false. Resharper is so nice, it tells me about that ;)

Reply:
https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs#L62

Reply:
What I meant was, do we need it if it never changes? :)

Reply:
Oh, sorry. Now i get your question.
It's not used in sql processing, but we store telemetry items in `CacheBasedOperationHolder` between start and stop. And We also use this class for http processing, where `isCustomCreated` was needed at least some time ago. 
This class requires us to have a tuple of `<DependencyTelemetry, bool>`.
It is a good idea to refactor it and get rid of the flag (I'm almost sure it's not used anymore), but I'd prefer not to do it in the scope of this PR.